nextcord/file.py

```diff
@@ -13,6 +13,12 @@ class File:
     r"""A parameter object used for :meth:`abc.Messageable.send`
     for sending file objects.
 
+    .. versionadded:: 2.4
```

Comment:
```suggestion
    .. versionchanged:: 2.5
```

Reply:
Isn't 2.4 the current version? I don't know how you guys version, but this isn't a breaking change so I think it's ok to be added to the current version.

Reply:
Correct 2.4 is the current version, which means it's already released. This feature at the earliest can get into 2.5

Reply:
We use a variant of [semantic versioning](https://semver.org/), which states that once a version is released, it cannot be modified.

Reply:
Understood. Thanks for the info.